trimesh/voxel.py

```diff
@@ -266,6 +291,62 @@ def voxelize_subdivide(mesh, pitch, max_iter=10):
 
     return voxels_sparse, origin_position
 
+def voxelize_subdivide_solid(mesh,pitch,max_iter=10):
+    voxels_sparse,origin_position=voxelize_subdivide(mesh,pitch,max_iter)
+    #create grid and mark inner voxels
+    max=voxels_sparse.max()+1
```

Comment:
`max` is a builtin function- I might rename this to `max_value` or something

Reply:
Thanks for the improvement. I'm not sure if we can vectorize the operation or not, I'll check it later. 
Just for reference, I tested the method with bunny.obj and buddha.obj, for a 64x64x64 grid, the former takes 0.36s (20% of total time) and the later takes 0.56s (8% of total time) to fill the inner voxels, which I think is reasonable now.